.flutter-plugins-dependencies

```diff
@@ -0,0 +1 @@
+{"_info":"// This is a generated file; do not edit or check into version control.","dependencyGraph":[{"name":"app_settings","dependencies":[]},{"name":"barcode_scan","dependencies":[]},{"name":"connectivity","dependencies":[]},{"name":"firebase_core","dependencies":[]},{"name":"firebase_database","dependencies":["firebase_core"]},{"name":"firebase_dynamic_links","dependencies":[]},{"name":"firebase_messaging","dependencies":[]},{"name":"flutter_secure_storage","dependencies":[]},{"name":"flutter_webview_plugin","dependencies":[]},{"name":"image_cropper","dependencies":[]},{"name":"image_picker","dependencies":[]},{"name":"keyboard_visibility","dependencies":[]},{"name":"local_auth","dependencies":[]},{"name":"package_info","dependencies":[]},{"name":"path_provider","dependencies":[]},{"name":"share_extend","dependencies":[]},{"name":"shared_preferences","dependencies":[]},{"name":"uni_links","dependencies":[]},{"name":"url_launcher","dependencies":[]}]}
```

Comment:
This file shouldn't be committed.

Reply:
[Deleted](https://github.com/breez/breezmobile/pull/258/commits/75d9ba9b09dda832a53b6a5caebdb0916c6cef0a)